sentry-spring-jakarta/src/test/kotlin/io/sentry/spring/jakarta/SentryCheckInAdviceTest.kt

```diff
@@ -157,6 +164,66 @@ class SentryCheckInAdviceTest {
         verify(hub, never()).popScope()
     }
 
+    @Test
+    fun `when @SentryCheckIn is passed a spring property it is resolved correctly`() {
+        val checkInId = SentryId()
+        val checkInCaptor = argumentCaptor<CheckIn>()
+        whenever(hub.captureCheckIn(checkInCaptor.capture())).thenReturn(checkInId)
+        val result = sampleServiceSpringProperties.hello()
+        assertEquals(1, result)
+        assertEquals(1, checkInCaptor.allValues.size)
+
+        val doneCheckIn = checkInCaptor.lastValue
+        assertEquals("mypropertycronslug", doneCheckIn.monitorSlug)
+        assertEquals(CheckInStatus.OK.apiName(), doneCheckIn.status)
+        assertNotNull(doneCheckIn.duration)
+
+        val order = inOrder(hub)
+        order.verify(hub).pushScope()
+        order.verify(hub).captureCheckIn(any())
+        order.verify(hub).popScope()
+    }
+
+    @Test
+    fun `when @SentryCheckIn is passed a spring property that does not exist, raw value is used`() {
+        val checkInId = SentryId()
+        val checkInCaptor = argumentCaptor<CheckIn>()
+        whenever(hub.captureCheckIn(checkInCaptor.capture())).thenReturn(checkInId)
+        val result = sampleServiceSpringProperties.helloUnresolvedProperty()
+        assertEquals(1, result)
+        assertEquals(1, checkInCaptor.allValues.size)
+
+        val doneCheckIn = checkInCaptor.lastValue
+        assertEquals("\${my.cron.wrong.property}", doneCheckIn.monitorSlug)
```

Comment:
Why is it `wrong.property`? Annotation below says `my.cron.unresolved.property`. Same question for the test below.

Reply:
Oversight on my part, after making property names more readable. This actually fails the test, will fix.